src/icons/two-toned/IconTaskApproval.tsx

```diff
@@ -11,6 +11,7 @@ const IconTaskApproval = ({ className = '', height = 24, title, width = 24 }: Tw
         title={title}
         viewBox="0 0 32 32"
         width={width}
+        tabIndex={0}
```

Comment:
this makes all usages of this icon tabbable, is that what we want? for example, this is used in task service client and EUA

Reply:
It is intended to be tabbable since if it is not, it won't be visible for screen reader. Those Icons are also used in content-sidebar in comment section which needs to be accessible.

Reply:
the screen reader can still read aria-labels even if the icon itself is not explicitly tabbable. I don't think we need to make the icon focusable otherwise we'd have accessibility issues with every icon used in BUIE since none of them currently have `tabIndex`, don't you think?

for example, we used the `title` prop to resolve accessibility issues for the "Global" icon used on Avatars:
https://github.com/box/box-ui-elements/blob/69293d8903f17d136aa567fd674e2fe263873d86/src/components/avatar/Avatar.tsx#L99

and this is read by screen readers without using tabIndex:
<img width="648" alt="Screenshot 2023-09-25 at 2 06 39 PM" src="https://github.com/box/box-ui-elements/assets/7311041/1d14a59e-a050-474c-9f30-9ac9f3b80a9d">

Reply:
this change would make all icons, regardless of it's usage, tabbable

e.g. if this icon is used inside of a button, such as for an IconButton, the component would now focus twice: once for the button focus and once for the icon focus. that doesn't seem right from a user experience perspective

Reply:
You are right, accessibility is working also when only title is passed. I did test it wrong previously so tabIndex is deleted